draft-ietf-rats-corim.md

```diff
@@ -490,8 +490,7 @@ The following describes each child item of this group.
 # Concise Module Identifier (CoMID) {#sec-comid}
 
 A CoMID tag contains information about hardware, firmware, or module composition that is provisioned
-in a Verifier. Each CoMID has a unique identifier that is used to identify tag instances unambigously
-with in the Verifier.
+in a Verifier. Each CoMID has a unique identifier (tag-id) that is used to identify tag instances unambigously in a Verifier. The tag-id is also used as a means to reference a CoMID from other tag, for example a CoBOM tag.
```

Comment:
suggest: 
"Each CoMID has a unique identifier (tag-id) that is used to unambiguously identify CoMID tags.
The `tag-id` may be used to cross reference CoMID tags from other manifest files and tags within the same manifest, such as a CoBOM, CoSWID, or TA-Store."